zerver/management/commands/send_custom_email.py

```diff
@@ -79,6 +85,15 @@ def handle(self, *args: Any, **options: str) -> None:
                 enable_marketing_emails=True,
                 long_term_idle=False,
             ).distinct("delivery_email")
+        elif options["remote_servers"]:
+            from zilencer.models import RemoteZulipServer
+
+            # TODO: Make this filter for deactivated=False once we add
+            # that to the data model.
+            target_emails = list(
+                set(RemoteZulipServer.objects.all().values_list("contact_email", flat=True))
+            )
+            users = []
```

Comment:
Pull this out to line 72, so that it's more guaranteed to be defined, no matter how the if/elsif clauses get shuffled or added to?